src/waveform/waveformmarklabel.cpp

```diff
@@ -47,9 +47,9 @@ void WaveformMarkLabel::prerender(QPointF bottomLeft, QPixmap icon, QString text
 
     QPainter painter(&m_pixmap);
 
-    painter.setPen(backgroundColor);
+    painter.setPen(QColor(Qt::transparent));
     painter.setBrush(QBrush(backgroundColor));
-    painter.drawRoundedRect(0, 0, pixmapRect.width(), pixmapRect.height(), 5.0, 5.0);
+    painter.drawRoundedRect(0, 0, pixmapRect.width(), pixmapRect.height(), 2.0, 2.0);
```

Comment:
Why was this changed?

Reply:
Because with 5.0, the labels are too much rounded, almost a circle in case of a single character. 
See: ![grafik](https://user-images.githubusercontent.com/1777442/65082039-457a8780-d9a5-11e9-89dd-f4741a1ff128.png)